source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -308,6 +309,14 @@ public static void setSelectAction(RenderedAdaptiveCard renderedCard, BaseAction
             view.setOnClickListener(new BaseActionElementRenderer.SelectActionOnClickListener(renderedCard, selectAction, cardActionHandler));
 
             applyTitleAndTooltip(selectAction, view);
+
+            if (view instanceof ViewGroup) {
+                ViewGroup group = (ViewGroup) view;
+
+                if (group.getChildCount() == 1 && group.getChildAt(0) instanceof TextView) {
```

Comment:
this seems to only work on TextView. ActionButton that's disabled should not be focusable? or the Image without SelectActions shouldn't be focusable, either. On iOS, there is a flag for user interactivity. Can you check if a view is interactive, then set the focusable state.

Reply:
Good call. This change is actually here - https://github.com/microsoft/AdaptiveCards/pull/8025.

The commit order got messed up. I'll make the fix there and merge that PR in first :)